assets/messages.yml

```
@@ -1039,6 +1039,19 @@ messages:
       shortcut: qul
       message: "%quick_links%"
       fillname: []
+  removed-version:
+    - project:
+        - mc
+        - mce
+        - mcl
+        - mcpe
```

Maybe it would be good to cover all projects which have affected versions because it appears Arisa is also running for all projects.
```suggestion
        - bds
        - mc
        - mcd
        - mce
        - mcl
        - mcpe
        - realms
```